pkgs/applications/networking/ssb-patchwork/default.nix

```diff
@@ -47,7 +47,7 @@ in
       '';
       homepage = "https://www.scuttlebutt.nz/";
       license = licenses.agpl3;
-      maintainers = with maintainers; [ asymmetric ninjatrappeur ];
+      maintainers = with maintainers; [ asymmetric ninjatrappeur cyplo ];
```

Comment:
@cyplo are you aware this is the last release there will ever be?

Reply:
not sure tbh, it's latest as of now;

are you okay with me being added to maintainers here ?

Reply:
> not sure tbh, it's latest as of now;

Here's the info: https://github.com/ssbc/patchwork/releases/tag/v3.18.1

> are you okay with me being added to maintainers here ?

sure :)